extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -113,43 +119,69 @@ export class DoubleclickA4aEligibility {
     return googleCdnProxyRegex.test(win.location.origin);
   }
 
-  /** Whether Fast Fetch is enabled
+  /**
+   * Attempts all unconditioned experiment selection.
    * @param {!Window} win
    * @param {!Element} element
-   * @param {!boolean} useRemoteHtml
-   * @return {boolean}
    */
-  isA4aEnabled(win, element, useRemoteHtml) {
-    let experimentId = this.maybeSelectExperiment(
-        win, element,[DOUBLECLICK_UNCONDITIONED_EXPERIMENTS.FF_CANONICAL_CTL,
+  unconditionedExperimentSelection(win, element) {
+    this.selectAndSetUnconditionedExp(
+        win, element,
+        [DOUBLECLICK_UNCONDITIONED_EXPERIMENTS.FF_CANONICAL_CTL,
           DOUBLECLICK_UNCONDITIONED_EXPERIMENTS.FF_CANONICAL_EXP],
-        DFP_UNCONDITIONED_CANONICAL_FF_EXPERIMENT_NAME);
+        UNCONDITIONED_CANONICAL_FF_EXPERIMENT_NAME);
+
+    this.selectAndSetUnconditionedExp(
+        win, element,
+        [DOUBLECLICK_UNCONDITIONED_EXPERIMENTS.IDENTITY_CONTROL,
+          DOUBLECLICK_UNCONDITIONED_EXPERIMENTS.IDENTITY_EXPERIMENT],
+        UNCONDITIONED_IDENTITY_EXPERIMENT_NAME);
+  }
+
+  /**
+   * Attempts to select into experiment and forces branch if selected.
+   * @param {!Window} win
+   * @param {!Element} element
+   * @param {!Array<string>} branches
+   * @param {!string} expName
+   */
+  selectAndSetUnconditionedExp(win, element, branches, expName) {
+    const experimentId = this.maybeSelectExperiment(
+        win, element, branches, expName);
     if (!!experimentId) {
       addExperimentIdToElement(experimentId, element);
-      forceExperimentBranch(
-          win, DFP_CANONICAL_FF_EXPERIMENT_NAME, experimentId);
+      forceExperimentBranch(win, expName, experimentId);
     }
+  }
 
+  /** Whether Fast Fetch is enabled
+   * @param {!Window} win
+   * @param {!Element} element
+   * @param {!boolean} useRemoteHtml
```

Comment:
Hate to stickle, but we should keep the type declarations consistent: either `!boolean` or `boolean`, with the latter preferred. Same with the `!string`'s in the new doc strings you added.